pom.xml

```diff
@@ -512,6 +512,36 @@
         </pluginManagement>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.3</version>
+                <configuration>
+                    <compilerId>javac-with-errorprone</compilerId>
+                    <compilerArgs>
+                       <arg>-Xep:JUnit4TestNotRun:WARN</arg>
```

Comment:
I had intended to leave all error checks at their default settings; however, this check proved to be too annoying to resolve. Error Prone expects ignored unit tests to have both the `@Ignore` and the `@Test` annotation. In many of our ignored cases, we only have the `@Ignore` annotation. As the test still does not run and the intent is clear, it seems overkill to update the several test classes in the system that are incorrect in this small way just to satisfy the tool.